test/domains/compilerErrors/notsupportedBulkAdd-ok.chpl

```diff
@@ -0,0 +1,14 @@
+use notsupportedCommon;
```

Comment:
Test name: Maybe rename to "BulkAdd-ok.chpl"?

Reply:
I chose `notsupported` as a common prefix for all these tests, to emphasize their similarity. Nevermind the grammar. :)